scheds/rust/scx_layered/src/main.rs

```diff
@@ -86,7 +86,7 @@ const NR_LLC_LSTATS: usize = bpf_intf::llc_layer_stat_id_NR_LLC_LSTATS as usize;
 const NR_LAYER_MATCH_KINDS: usize = bpf_intf::layer_match_kind_NR_LAYER_MATCH_KINDS as usize;
 
 lazy_static! {
-    static ref USAGE_DECAY: f64 = 0.5f64.powf(1.0 / USAGE_HALF_LIFE_F64);
+    static ref USAGE_DECAY: f64 = 0.2f64.powf(1.0 / USAGE_HALF_LIFE_F64);
```

Comment:
pulled out calculator btw, I think `f64.powf(1.0 / USAGE_HALF_LIFE_F64)` is maybe always just 1 btw.

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=172222a94132f6336d0fedb6db887745

Doesn't look like 1 to me.

Reply:
I missed the divide by 1 billion while grepping for the consts involved, but tl;dr, as that is, it's a complicated way of writing `0.5*1^(1/(100 million/1 billion))`

Reply:
Yeah, the HALF_LIFE_F64 has a pretty bad name given it's not just HALF_LIFE as f64! Would probably be better if they were named `HALF_LIFE_NS` and `HALF_LIFE_S_F64` or something.